biomage/utils/db.py

```diff
@@ -0,0 +1,37 @@
+import json
+
+from ..rds.run import run_rds_command
+
+
+def _process_query_output(query_result):
+    json_text = (
+        query_result.replace("+", "")
+        .split("\n", 2)[2]
+        .replace("\n", "")
+        .replace("(1 row)", "")
+        .strip()
+    )
```

Comment:
This the same thing as in upload.py? DId you make upload.py use it too? Let's try to avoid duplicate code

Reply:
no, the ones in `upload.py` is already there. It's already a duplicate of `download.py`.

Reply:
maybe it's time to add it to a file in utils?

Reply:
I'm refactoring this

Reply:
I've refactored `download.py`, but I can't test for `upload.py` because of permissions (see [discusion](https://this-is-biomage.slack.com/archives/C015CHSUDRU/p1668768076538709)). I don't want to push untested code, so I'm doing the refactor for `download.py`.